david_vallejos/space_challenge/src/main/java/com/tech_group/Main.java

```diff
@@ -0,0 +1,40 @@
+package main.java.com.tech_group;
+
+import java.text.NumberFormat;
+import java.util.ArrayList;
+
+/**
+ * Class to start simulations.
+ */
+public class Main {
+
+    public static void main(String[] args) {
+
+        final String filePathPhase1 = "F:\\SDF\\TechGroup1-20\\david_vallejos\\space_challenge\\src\\Phase-1.txt";
+        final String filePathPhase2 = "F:\\SDF\\TechGroup1-20\\david_vallejos\\space_challenge\\src\\Phase-2.txt";
+        final NumberFormat format = NumberFormat.getCurrencyInstance();
+        Simulation firstSimulation = new Simulation();
+
+
+        ArrayList<Item> itemsPhase1 = firstSimulation.loadItems(filePathPhase1);
+
```

Comment:
When declaring a collection please use the abstract class, this is a good practice.
Example:
```java
List<Item> itemsPhase1  = new ArrayList<>();
```
Please **update all variables** according to what was explained.

Reply:
Done, I updated all variables